masonry/src/theme.rs

```diff
@@ -87,17 +88,37 @@ pub fn default_property_set() -> DefaultProperties {
     properties.insert::<TextInput, _>(BorderWidth {
         width: BORDER_WIDTH,
     });
-    properties.insert::<TextInput, _>(PlaceholderColor::new(PLACEHOLDER_COLOR));
-
     properties.insert::<TextInput, _>(BorderColor { color: ZYNC_600 });
+    properties.insert::<TextInput, _>(PlaceholderColor::new(PLACEHOLDER_COLOR));
+    properties.insert::<TextInput, _>(CaretColor { color: TEXT_COLOR });
+    properties.insert::<TextInput, _>(SelectionColor {
+        color: ACCENT_COLOR,
+    });
+    properties.insert::<TextInput, _>(UnfocusedSelectionColor(SelectionColor {
+        color: DISABLED_TEXT_COLOR,
+    }));
 
     // TextArea
     properties.insert::<TextArea<false>, _>(ContentColor::new(TEXT_COLOR));
     properties
         .insert::<TextArea<false>, _>(DisabledContentColor(ContentColor::new(DISABLED_TEXT_COLOR)));
+    properties.insert::<TextArea<false>, _>(CaretColor { color: TEXT_COLOR });
+    properties.insert::<TextArea<false>, _>(SelectionColor {
+        color: ACCENT_COLOR,
+    });
+    properties.insert::<TextArea<false>, _>(UnfocusedSelectionColor(SelectionColor {
+        color: DISABLED_TEXT_COLOR,
+    }));
     properties.insert::<TextArea<true>, _>(ContentColor::new(TEXT_COLOR));
     properties
         .insert::<TextArea<true>, _>(DisabledContentColor(ContentColor::new(DISABLED_TEXT_COLOR)));
+    properties.insert::<TextArea<true>, _>(CaretColor { color: TEXT_COLOR });
+    properties.insert::<TextArea<true>, _>(SelectionColor {
+        color: ACCENT_COLOR,
+    });
+    properties.insert::<TextArea<true>, _>(UnfocusedSelectionColor(SelectionColor {
+        color: DISABLED_TEXT_COLOR,
+    }));
```

Comment:
Do these defaults not need to be on `TextInput`, not `TextArea`? (as they will get immediately overwritten to the static defaults by the `WidgetAdded` code)

Reply:
I thought that `TextArea` could be used independently of `TextInput`, like a multiline version.